### DISCUSSION TITLE
Enable specific days

### DISCUSSION THREAD

**Discussion by landsman** [gpbl/react-day-picker#1757] [Support]
This is follow-up to already existing: https://github.com/gpbl/react-day-picker/discussions/1584

When you have turn on property `showOutsideDays` there is no easy way how to achieve this design from outside I think.

Hard way would be to generate "occupied days" for specified month with some "drop off" before and after, which is kinda overkill.
So I can imagine some kind of property to switch this logic from "disabled" to "allowed". 

What do you think?

**Comment by gpbl** [comment]
@landsman Would you replicate your problem in https://codesandbox.io/s/react-daypicker-vl4ry? Thanks!

**Comment by landsman** [comment]
I tried and found bug in my code, sorry for that!
